### PR TITLE
adds 'intro to greg' post

### PR DESCRIPTION
**Pull request by gregsdennis**
I've joined Postman, too!

Depends on #13

### CODE REVIEW

**Comment by Julian** [comment]
Nice! LGTM I think, post looks good! No nitpicks even.

**Review by Relequestual — CHANGES_REQUESTED** [review]
Can you change your images to be webp format please? They are super speedy =]
(https://github.com/json-schema-org/blog#requirements)

- [ ] Image format
- [ ] Update link to Julians article


Then I think it's good to go!

File: pages/posts/and-then-there-were-three.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+Hi, everybody!
+
+I'm Greg Dennis.  I'm the author of [`json-everything`](https://github.com/gregsdennis/json-everything), which aims to be an all-encompassing tool suite for JSON needs in .Net, including JSON Path, JSON Patch, and (of course) JSON Schema, amongst others.  I'm also responsible for driving and authoring the output requirements in the JSON Schema specification.
+
+Piggybacking on [Julian's news](hello-world-hello-postman), I'm also joining Postman to work on JSON Schema full time as a Specification & Tooling Architect!  This means that I will be working on updating the specification as well as helping define and implement validation and other JSON-Schema-adjacent tooling.  I also expect to be highly involved in the community by supporting other tooling implementors.
+
+## Being Involved
+
+JSON Schema has always been a passion project for me.  I didn't really see it at work\*, and I never had a problem for which JSON Schema was the solution.  Nevertheless, I recognized its potential that it _could_ be the solution other people needed, and I wanted to contribute by giving them an easy adoption path.
```
[the remark this answers is not in the document]
Comment (Relequestual) [reply pages/posts/and-then-there-were-three.md:25]: ooooh. I made an assumption. My bad. Withdrawn.

**Comment by gregsdennis** [comment]
Images fixed.  Link to Julian's post is correct as is.

**Review by Relequestual — APPROVED** [review]

**Pull request merged**